doc/sailfish-webview-webviewadditional.qdoc

```diff
@@ -0,0 +1,321 @@
+\l{WebView::scrollableOffset} x-value plus the (\l{WebView::resolution} scaled)
+\l{WebView::contentRect} width is approximately equal to or greater than the
+\l{WebView::scrollableSize} width.
+
+See also: \l{WebView::atXBeginning}, \l{WebView::scrollableOffset},
+\l{WebView::scrollableSize}, \l{WebView::contentRect}
+
+\section2 WebView::atYBeginning
+
+\c{bool}-type read-only property.
+
+Whether the viewport of the webview is scrolled to the top of the content.
+
+Effectively, the value of this property will be true if the
+\l{WebView::scrollableOffset} y-value is approximately zero.
+
+See also: \l{WebView::atYEnd}, \l{WebView::scrollableOffset}
+
+\section2 WebView::atYEnd
+
+\c{bool}-type read-only property.
+
+Whether the viewport of the webview is scrolled to the bottom of the
+content.
+
+Effectively, the value of this property will be true if the
+\l{WebView::scrollableOffset} y-value plus the (\l{WebView::resolution} scaled)
+\l{WebView::contentRect} height is approximately equal to or greater than the
+\l{WebView::scrollableSize} height.
+
+See also: \l{WebView::atYBeginning}, \l{WebView::scrollableOffset},
+\l{WebView::scrollableSize}, \l{WebView::contentRect}
+
+\section2 WebView::dragging
+
+\c{bool}-type read-only property.
+
+Whether the user is currently dragging the scrollable area.
+
+\section2 WebView::moving
+
+\c{bool}-type read-only property.
+
+Whether the scrollable area is currently moving due to
+inertia from a previous drag gesture.
+
+\section2 WebView::pinching
+
+\c{bool}-type read-only property.
+
+Whether the user is currently performing a pinch gesture
+in order to zoom the web view.
+
+\section2 WebView::chrome
+
+\c{bool}-type property.
+
+Whether the chrome (i.e. user interface elements for navigation) is visible.
+
+By default, the chrome will be shown or hidden in response to certain
+gestures.  The client can override this behavior by setting the value
+manually.
+
+See also: \l{WebView::chromeGestureEnabled},
+\l{WebView::chromeGestureThreshold}
+
+\section2 WebView::chromeGestureEnabled
+
+\c{bool}-type property.
+
+Whether the gesture to show the navigation chrome is enabled.
+
+If true, chrome is set to false (hiding the navigation chrome) when
+a pan or flick gesture causes movement exceeding the chromeGestureThreshold,
+and automatically set back to true when a similar gesture in the opposite
+direction is detected.
+
+See also: \l{WebView::chrome}, \l{WebView::chromeGestureThreshold}
+
+\section2 WebView::chromeGestureThreshold
+
+\c{qreal}-type property.
+
+The movement threshold for detecting a chrome gesture.
+
+See also: \l{WebView::chromeGestureEnabled}, \l{WebView::chrome}
+
+\section2 WebView::desktopMode
+
+\c{bool}-type property.
+
+Whether the web view will attempt to load the desktop version of the
+site rather than the mobile version.
+
+If the request cannot be served, the previously loaded content will be
+displayed instead.
```

Comment:
As this really depends when the property is changed and in reality "previously loaded" triggers reload as wll... I'd maybe turn this into something like...

"It depends on the web service whether the desktop mode can be served or not"

Reply:
done